content/en/news/2019/announcing-1.4/upgrade-notes/index.md

```diff
@@ -22,6 +24,12 @@ If you depend on this behavior, there are a few options:
 
 See [Protocol Selection](/docs/ops/traffic-management/protocol-selection/) for more information about specifying the protocol of a port
 
+### Regex Engine Changes
+
+To prevent excessive resource consumption from large regular expressions, Envoy has moved to a new regular expression engine based on [`re2`](https://github.com/google/re2). Previously, `std::regex` was used. These two engines may have slightly different syntax; in particular, the regex fields are now limited to 100 bytes.
```

Comment:
```suggestion
To prevent excessive resource consumption from large regular expressions, Envoy has moved to a new regular expression engine based on [`re2`](https://github.com/google/re2). Previously, `std::regex` was used. These two engines may have slightly different syntax; in particular, the regex fields are now limited to 100 characters.
```